dotCMS/src/main/java/com/dotmarketing/common/util/SQLUtil.java

```diff
@@ -310,4 +316,134 @@ private static boolean isValidSQLCharacter (final char c) {
 		return Character.isLetterOrDigit(c) || '-' == c || '_' == c;
 	} // isValidSQLCharacter.
 
+
+	//H2 Upsert Example:
+	//MERGE INTO table (columns) KEY (conditionalColumn) VALUES (values)
+	private final static String H2_UPSERT_QUERY =
+			"MERGE INTO %s (%s) KEY (%s) VALUES (%s) ";
+
+	// Postgres Upsert Example:
+	// INSERT INTO table (columns) VALUES (values)
+	// ON CONFLICT (conditionalColumn) DO UPDATE SET column1=value1, column2=value2, etc...
+	private final static String POSTGRES_UPSERT_QUERY =
+			"INSERT INTO %s (%s) "
+			+ "VALUES (%s) ON CONFLICT (%s) "
+			+ "DO UPDATE SET %s";
+
+	// MySQL Upsert Example:
+	// INSERT INTO table (columns) VALUES (values)
+	// ON DUPLICATE KEY UPDATE column1=value1, column2=value2, etc...
+	private final static String MYSQL_UPSERT_QUERY =
+			"INSERT INTO %s (%s) "
+			+ "VALUES (%s) ON DUPLICATE KEY "
+			+ "UPDATE %s";
+
+	// MSSQL Server Upsert Example:
+	// MERGE table WITH (HOLDLOCK) AS [Target] USING
+	// (SELECT conditionalValue AS conditionalColumn) AS [Source] ON [Target].conditionalColumn = [Source].conditionalColumn
+	// WHEN MATCHED THEN UPDATE SET column1=value1, column2=value2, etc...
+	// WHEN NOT MATCHED THEN INSERT (columns) VALUES (values);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)